Include/abstract.h

```diff
@@ -397,6 +397,13 @@ PyAPI_FUNC(int) PyIter_Check(PyObject *);
    This function always succeeds. */
 PyAPI_FUNC(int) PyAIter_Check(PyObject *);
 
+/* Takes an iterator object and calls its tp_iternext slot,
+   setting *item to the next value, or to NULL if the iterator
+   is exhausted.
+
+   returns 0 on success and -1 on error. */
```

Comment:
```suggestion
   Returns 0 on success and -1 on error. */
```

Reply:
Or even better with the imperative mood.

```suggestion
   Return 0 on success and -1 on error. */
```